app/views/layouts/application.html.erb

```diff
@@ -53,12 +53,16 @@
         </form> -->
         <ul class="nav navbar-nav navbar-right">
           <% if current_user %>
+            <% if current_user.subscribed %>
+              <li><%= link_to current_user.email, account_path %></li>
+            <% else %>
+              <li><a href="/account">Subscribe to Pro</a></li>
+            <% end %>
+              <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
```

Comment:
So the typical indentation I've seen is, if you have an opening and closing tag, then everything in between gets indented another line.  So the two `li`s within the if/else/end block are indented an extra line in, but the one after (not between the opening if and closing end) does not have the extra indent.